forms-flow-bpm/LICENSE

```diff
@@ -0,0 +1,203 @@
+Apache License
```

Comment:
What is this new file

Reply:
It's the LICENSE which was added by our lawyer in forms-flow-ai